spec/classes/mysql_backup_xtrabackup_spec.rb

```diff
@@ -42,8 +42,6 @@ class { 'mysql::server': }
                       'percona-xtrabackup-24'
                     elsif Puppet::Util::Package.versioncmp(facts[:os]['release']['major'], '7') >= 0
```

Comment:
This should just be an else now

Reply:
My thought is that an if/elsif should always end in an else, so you have the if OS is 8 or above  followed by the else that catches 7.

Reply:
Yeah, my bad. I completely agree. I misunderstood that but its clear now. Thanks for pointing out @david22swan